AppRate-Demo/proguard-rules.pro

```diff
@@ -1,17 +0,0 @@
-# Add project specific ProGuard rules here.
```

Comment:
Just adding "config module specific ProGuard rules here"  should be enough, remove other content